sorting/selectionsort.py

```diff
@@ -0,0 +1,31 @@
+# Selection sort in Python
```

Comment:
looks good! could you take up more issues?

Reply:
Yes I can👍